modules/nw-udn-cr.adoc

```diff
@@ -38,7 +38,7 @@ spec:
 <1> Name of your `UserDefinedNetwork` resource. This should not be `default` or duplicate any global namespaces created by the Cluster Network Operator (CNO).
 <2> The `topology` field describes the network configuration; accepted values are `Layer2` and `Layer3`. Specifying a `Layer2` topology type creates one logical switch that is shared by all nodes.
 <3> This field specifies the topology configuration. It can be `layer2` or `layer3`.
-<4> Specifies `Primary` or `Secondary`. `Primary` is the only `role` specification supported in 4.17.
+<4> Specifies `Primary` or `Secondary`.
```

Comment:
It's probably fine as is, but i'd consider adding a noun after these values for clarity:
```suggestion
<4> Specifies a `Primary` or `Secondary` role.
```

Up to you, and of course feel free to choose any noun you find best for this description.